README.md

```diff
@@ -46,25 +46,17 @@ PMWatch is dependent on [libipmctl](https://github.com/intel/ipmctl).
 Perform the following steps to install the tool dependencies.
 
 #### Fedora
-> $ yum install daxctl-devel.x86_64 ndctl-devel.x86_64 libipmctl-devel.x86_64 libsafec-devel.x86_64
```

Comment:
Is ipmctl no longer dependent on libsafec?

Reply:
Since version 2.0, no longer.  And for version 1, yum/apt will pull it if needed.

Reply:
If I remember correctly, only Fedora had libsafec in its repo. I added the URLs as it was not available/easy to install in Ubuntu, debian, suse systems. If the situation persists, I prefer to keep the URL information.

Reply:
Every distribution that has ipmctl has safeclib (as it's a dependency).